plan/logical_plan_builder.go

```diff
@@ -431,6 +431,11 @@ func (b *planBuilder) buildProjection(p LogicalPlan, fields []*ast.SelectField,
 				}
 			}
 		}
+
+		if colName.O == "\\N" {
```

Comment:
Could you add a test case for this?
You can add a test case like `select \N;` and check the column name and row value.